### ISSUE TITLE
Feature request: add dual ym2151 support for sharp x1

### ISSUE THREAD

**Issue by denjhang** [mamedev/mame#8710]
There is a sound driver named NRTDRV, which can call two YM2151 (OPM) and one YM2149 (PSG). Some specific details need to be done on the sharp x1 physical machine

http://nrtdrv.sakura.ne.jp/index.cgi?page=FM%B2%BB%B8%BB%A5%DC%A1%BC%A5%C9I%2FO%A5%A2%A5%C9%A5%EC%A5%B9%CA%D1%B9%B9%CA%FD%CB%A1
http://nrtdrv.sakura.ne.jp/index.cgi?page=NRDSEL

**Comment by rb6502** [comment]
If I understand correctly, the YM2151 is on an expansion card, and it's possible to add a second modified card with the second 2151.  We don't support any kind of slots for the X1 series yet, which would be the primary blocker for this feature.

**Comment by angelosa** [comment]
Correct, the technical term for this specific board is _CZ-8BS1_, while the bus maps boards codenamed CZ-* specifically for X1:
http://retropc.net/ohishi/museum/cznum.htm
https://www.musee-des-jeux-video.com/screenshots/Sharp%20X1/doc/Sharp%20X1%20Series%20catalog%20(Japan).pdf
